app/src/main/java/com/example/cocktails/CocktailsFragment.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.cocktails
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+
+class CocktailsFragment : Fragment() {
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+    }
+
+    override fun onCreateView(
```

Comment:
We can replace onCreateView with class Cocktails : Fragment(R.layout.fragment_cocktails) {}